release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -682,6 +710,20 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* In {product-title} 4.6, the resource metrics API server provides support for custom metrics. The resource metrics API server does not implement the OpenAPI specification, and the following messages are sent to the `kube-apiserver` logs every 60 seconds:
```

Comment:
from https://bugzilla.redhat.com/show_bug.cgi?id=1819053#c1
the message frequency is every 120 seconds

Reply:
Thank you for highlighting this. I have now removed 'every 60 seconds' as advised.